<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wizard</class>
 <widget class="QWizard" name="Wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>488</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>366</width>
    <height>488</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Unvanquished Installer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../.designer/.designer/Pictures/picture_1.png</normaloff>../.designer/.designer/Pictures/picture_1.png</iconset>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhUrlCharactersOnly</set>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::NoCancelButton</set>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>Select the Maps To Include</string>
   </property>
   <property name="subTitle">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="mapsToIncludeLabel">
      <property name="text">
       <string>Maps to include:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="allCheckBox">
      <property name="text">
       <string>All</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QTableWidget" name="mapsToIncludeTableWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>temp1</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="browseButton">
      <property name="text">
       <string>Browse</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="directoryToInstallInLabel">
      <property name="text">
       <string>Directory to install in:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="directoryToInstallInLineEdit">
      <property name="inputMethodHints">
       <set>Qt::ImhNoPredictiveText|Qt::ImhUrlCharactersOnly</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLabel" name="sizeOfRequiredFilesLabel">
      <property name="text">
       <string>Size of files needed by Unvanquished: {}</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QLabel" name="sizeOfSelectedMapsLabel">
      <property name="text">
       <string>Size of maps selected: {}</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QLabel" name="totalDownloadSizeLabel">
      <property name="text">
       <string>Total download size: {}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="16" column="0">
     <widget class="QCheckBox" name="fileInfoCheckBox">
      <property name="text">
       <string>Show File Information Table</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="totalETA">
      <property name="text">
       <string>Unknown</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="totalFileProgress">
      <property name="text">
       <string>45th file out of 300 files</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="currentFileETA">
      <property name="text">
       <string>Unknown</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="currentDownloadSpeedLabel">
      <property name="text">
       <string>Connecting...</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="3">
     <widget class="QTableWidget" name="fileInfoTableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>2</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>temp</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>temp2</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Version</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>The Total Data on The Internet</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>N/A</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>Looks like nothin'</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>Dumdadumdadumda</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>3.1.4.1.5.1.6.9.2.6</string>
       </property>
       <property name="background">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>142</red>
          <green>255</green>
          <blue>132</blue>
         </color>
        </brush>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>21309123908 ZB</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QProgressBar" name="currentFileProgressBar"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="currentFileName">
      <property name="text">
       <string>ATCS...</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="currentFileDownloadProgress">
      <property name="text">
       <string>43 MiB / 54 MiB</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="3">
     <widget class="QProgressBar" name="totalProgressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QLabel" name="totalDownloadProgress">
      <property name="text">
       <string>1 GiB / 2 GiB</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage3">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="displayUnvInAppMenu">
      <property name="text">
       <string>Display Unvanquished in the application menu.</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="displayUnvInDesktop">
      <property name="text">
       <string>Display Unvanquished in the desktop.</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="runUnvAfterFinish">
      <property name="text">
       <string>Run Unvanquished after finish.</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <tabstops>
  <tabstop>mapsToIncludeTableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
